Kickstarter-iOS/AppDelegateViewModel.swift

```diff
@@ -126,12 +126,6 @@ public protocol AppDelegateViewModelOutputs {
   /// Emits when the root view controller should navigate to activity.
   var goToActivity: Signal<(), Never> { get }
 
-  /// Emits when application should navigate to the creator's message thread
-  var goToCreatorMessageThread: Signal<(Param, MessageThread), Never> { get }
-
-  /// Emits when the root view controller should navigate to the creator dashboard.
-  var goToDashboard: Signal<Param?, Never> { get }
-
   /// Emits when the root view controller should navigate to the creator dashboard.
   var goToDiscovery: Signal<DiscoveryParams?, Never> { get }
```

Comment:
nit: the comment here is misleading. it should probably say something like 
```suggestion
  /// Emits when the root view controller should navigate to the discovery screen.
  var goToDiscovery: Signal<DiscoveryParams?, Never> { get }
```

Reply:
Whoops, thanks for catching that!